mycity/mycity/mycity_controller.py

```diff
@@ -60,6 +65,18 @@ def execute_request(mycity_request):
     #         "amzn1.echo-sdk-ams.app.[unique-value-here]"):
     #     raise ValueError("Invalid Application ID")
 
+    # First check if user's device is in Boston
+    if mycity_request.geolocation_coordinates is not None:
+        lat = mycity_request.geolocation_coordinates['latitudeInDegrees']
+        long = mycity_request.geolocation_coordinates['longitudeInDegrees']
+        location = gis_utils.reverse_geocode_addr([long, lat])
+        if location['address']['City'] != 'Boston' or \
+                location['address']['Region'] != 'Massachusetts':
+            mycity_response = MyCityResponseDataModel()
+            mycity_response.output_speech = NOT_IN_BOSTON_SPEECH
+            # TODO: End Session
```

Comment:
Please do this TODO. Not ending the session appropriately has caused issues with getting the app approved.